packages/docs/docs/config/README.md

```diff
@@ -222,7 +222,12 @@ The key and value pair will be added to `<a>` tags that points to an external li
 - Type: `Object`
 - Default: `{ includeLevel: [2, 3] }`
 
-Options for [markdown-it-table-of-contents](https://github.com/Oktavilla/markdown-it-table-of-contents). (Note: prefer `markdown.slugify` if you want to customize header ids.)
+This attribute will control the behaviour of `[[TOC]]`. We design this interface as close as possible to the options of [markdown-it-table-of-contents](https://github.com/Oktavilla/markdown-it-table-of-contents). However, there are some little differences:
+
+1. `slugify`, `format` and `forceFullToc` are not supported at present.
```

Comment:
We don't need to support `slugify` individually for this TOC component or the link would go the 404.

Unsupported options aren't recommended in the docs, it should be mentioned as a breaking change in the changelog.